src/github.com/getlantern/flashlight/genconfig/cloud.yaml.tmpl

```diff
@@ -7,7 +7,7 @@ client:
   frontedservers: []
   chainedservers: {{range .fallbacks}}
     fallback-{{.ip}}:
-      addr: {{.ip}}:443
+      addr: {{.ip}}
```

Comment:
Yeah it works for whatever reason @fffw -- maybe it looks for colon followed by some whitespace?